app/less/themes/dark.less

```diff
@@ -0,0 +1,112 @@
+/* Dark theme common css */
```

Comment:
Maybe a general improvement to the codebase (since we're working on themes) is to add a colours file, that way we can be consistent in how we'er referencing them (e.g. codebase includes both #fff and #ffffff) - this also paves the way for more themes :)

Reply:
Looks like we could extend over here.. https://github.com/zhukov/webogram/blob/master/app/less/lib/mixins.less

Reply:
I won't go too much into details in this PR as I don't know the code base at all